plugins/janus_sip.c

```diff
@@ -2986,37 +3064,54 @@ static void *janus_sip_handler(void *data) {
 			}
 
 			session->account.registration_status = janus_sip_registration_status_registering;
-			if(!refresh && session->stack == NULL) {
+			if (!refresh) {
```

Comment:
Why have you removed this check?

Reply:
stack is created in janus_sip_create_session. So, it cannot be NULL here.